pkg/storage/secret/store.go

```diff
@@ -558,8 +588,11 @@ func (s *secureStore) Delete(ctx context.Context, ns string, name string) (*secr
 		return nil, false, err
 	}
 
+	// TODO: maybe we always return `true`, when there's no error?
 	deleted := false
+
 	err = s.db.WithTransaction(ctx, func(tx *session.SessionTx) error {
+		// TODO: this should be in a template as well?
```

Comment:
it's simple enough to be in the code, up to us if we feel like it's easier to modify templates;

we could also maybe add more safeguards to the query like namespace